sdk/deviceupdate/Azure.IoT.DeviceUpdate/README.md

```diff
@@ -48,12 +47,11 @@ All Device Update for IoT Hub service operations will throw a RequestFailedExcep
 
 For example, if you use the `GetUpdateAsync` operation and the model you are looking for doesn't exist, you can catch that specific [HttpStatusCode](https://docs.microsoft.com/dotnet/api/system.net.httpstatuscode?view=netcore-3.1) to decide the operation that follows in that case.
 
-```csharp
+```c#
```

Comment:
We use ```csharp here by convention in the README's, except for snippets.

Reply:
The issue to track: https://github.com/Azure/azure-sdk-for-net/issues/26455